### PR TITLE
added door positions

### PR DESCRIPTION
**Pull request by XeAri**

### CODE REVIEW

**Comment by NotAdam** [comment]
PR this to ufx/SaintCoinach if you haven't already - we can't maintain manual additions to generated data

File: src/common/Network/PacketDef/Ipcs.h [hunk 1]
```diff
@@ -216,7 +216,9 @@ namespace Core::Network::Packets
 
     // Unknown IPC types that still need to be sent
     // TODO: figure all these out properly
+    // Quests related op codes
     IPCTYPE_UNK_320 = 0x0249, // updated 4.4
+    IPCTYPE_UPDATE_320 = 0x024A, // updated 4.4
```
Comment (NotAdam) [inline src/common/Network/PacketDef/Ipcs.h]: what is this?

File: src/common/Network/PacketDef/Ipcs.h [hunk 2]
```diff
@@ -216,7 +216,9 @@ namespace Core::Network::Packets
 
     // Unknown IPC types that still need to be sent
     // TODO: figure all these out properly
+    // Quests related op codes
```
Comment (NotAdam) [inline src/common/Network/PacketDef/Ipcs.h]: are you sure? do you have any examples where retail sends these packets alongside quest updates

**Comment by takhlaq** [comment]
there's a pr for this https://github.com/ufx/SaintCoinach/pull/197

File: src/servers/sapphire_zone/Zone/House.cpp [hunk 3]
```diff
@@ -122,6 +132,11 @@ uint32_t Core::House::getHouseId() const
   return m_houseId;
 }
 
+Core::Common::FFXIVARR_POSITION3 Core::House::getDoorPosition() const
```
Comment (NotAdam) [inline src/servers/sapphire_zone/Zone/House.cpp]: I just checked and these positions don't match all the doors, plot 3 goblet is an example of this where its inside the front wall to the right

Comment (NotAdam) [reply src/servers/sapphire_zone/Zone/House.cpp]: it looks like it's the world position of where the icon is on the map

Comment (SapphireMordred) [reply src/servers/sapphire_zone/Zone/House.cpp]: Which the exd name kindof gives away ;)

**Pull request closed without merging**